### ISSUE TITLE
[QBFT] Deploying a Besu testnet, blocks are not produced.

### ISSUE THREAD

**Issue by dottscarlato** [hyperledger/besu#3946]
Hi there,

I am trying to run a QBFT private network, following this [guide](https://besu.hyperledger.org/en/stable/Tutorials/Private-Network/Create-QBFT-Network/#create-a-private-network-using-the-qbft-proof-of-authority-consensus-protocol).

Given that the `bin/besu` directory (as indicated [here](https://besu.hyperledger.org/en/stable/HowTo/Get-Started/Installation-Options/Install-Binaries/#linux-unix)) is not present in recent versions, I am using the `21.10.4` version.

I performed these steps:

```
wget https://hyperledger.jfrog.io/artifactory/besu-binaries/besu/21.10.4/besu-21.10.4.tar.gz
tar -xvzf besu-21.10.4.tar.gz besu-21.10.4/
cd besu-21.10.4/
bin/besu --help

mkdir QBFT-Network/
mkdir QBFT-Network/Node-1/
mkdir QBFT-Network/Node-1/data
mkdir QBFT-Network/Node-2/
mkdir QBFT-Network/Node-2/data
mkdir QBFT-Network/Node-3/
mkdir QBFT-Network/Node-3/data
mkdir QBFT-Network/Node-4/
mkdir QBFT-Network/Node-4/data

cd QBFT-Network/
touch qbftConfigFile.json
# https://besu.hyperledger.org/en/stable/Tutorials/Private-Network/Create-QBFT-Network/#2-create-a-configuration-file

../bin/besu operator generate-blockchain-config --config-file=qbftConfigFile.json --to=networkFiles --private-key-file-name=key
cp networkFiles/genesis.json .

######### THIS PART I WAS MISSING ############
mv networkFiles/keys/0x1110a70d69cfca3541d8cc4f5834bb2e85580568/key* Node-1/data/
mv networkFiles/keys/0x3c4b27d054074285c7e8a49daf819fd556034ec8/key* Node-2/data/
mv networkFiles/keys/0x5cca249ba24166a8dff9a27d29bc142fbfb43c1c/key* Node-3/data/
mv networkFiles/keys/0x669b2284d610bd785354f4da25d205f10078504f/key* Node-4/data/

#cd /home/michelescarlato/quorum-test-network/network
#./stop.sh
cd
cd besu-21.10.4/QBFT-Network/Node-1

../../bin/besu --data-path=data --genesis-file=../genesis.json --rpc-http-enabled --rpc-http-api=ETH,NET,QBFT --host-allowlist="*" --rpc-http-cors-origins="all"

# in another shell

cd besu-21.10.4/QBFT-Network/Node-2

../../bin/besu --data-path=data --genesis-file=../genesis.json --bootnodes=enode://d9664997d905fc7d087ed9361f16ed8cb01dd244408666d74110fe4a554b7c3300155c85b8304fdad2a5f7028f4d7f5b65a46ae6a13ef4f840198bae171a4a2d@127.0.0.1:30303 --p2p-port=30304 --rpc-http-enabled --rpc-http-api=ETH,NET,QBFT --host-allowlist="*" --rpc-http-cors-origins="all" --rpc-http-port=8546

# in another shell

cd besu-21.10.4/QBFT-Network/Node-3

../../bin/besu --data-path=data --genesis-file=../genesis.json --bootnodes=enode://d9664997d905fc7d087ed9361f16ed8cb01dd244408666d74110fe4a554b7c3300155c85b8304fdad2a5f7028f4d7f5b65a46ae6a13ef4f840198bae171a4a2d@127.0.0.1:30303 --p2p-port=30305 --rpc-http-enabled --rpc-http-api=ETH,NET,QBFT --host-allowlist="*" --rpc-http-cors-origins="all" --rpc-http-port=8547

# in another shell

cd besu-21.10.4/QBFT-Network/Node-4
../../bin/besu --data-path=data --genesis-file=../genesis.json --bootnodes=enode://d9664997d905fc7d087ed9361f16ed8cb01dd244408666d74110fe4a554b7c3300155c85b8304fdad2a5f7028f4d7f5b65a46ae6a13ef4f840198bae171a4a2d@127.0.0.1:30303 --p2p-port=30306 --rpc-http-enabled --rpc-http-api=ETH,NET,QBFT --host-allowlist="*" --rpc-http-cors-origins="all" --rpc-http-port=8548

curl -X POST --data '{"jsonrpc":"2.0","method":"qbft_getValidatorsByBlockNumber","params":["latest"], "id":1}' localhost:8545
```

as indicated in the guide.

The nodes are producing the following output:

```
2022-06-08 14:20:24.479+02:00 | pool-8-thread-1 | INFO  | GasLimitRangeAndDeltaValidationRule | Invalid block header: basefee should not be present in a block without a base fee market
2022-06-08 14:20:24.479+02:00 | pool-8-thread-1 | INFO  | ProposalPayloadValidator | Invalid Proposal Payload: block did not pass validation.
2022-06-08 14:20:24.479+02:00 | pool-8-thread-1 | INFO  | ProposalValidator | Invalid Proposal Payload: invalid proposal payload in proposal message

```

Is there any setting that I am missing?

Thanks,
M

**Comment by siladu** [comment]
Hi @dottscarlato, see the reply here: https://github.com/hyperledger/besu/issues/3952#issuecomment-1153339532

**Comment by dottscarlato** [comment]
> Hi @dottscarlato, see the reply here: [#3952 (comment)](https://github.com/hyperledger/besu/issues/3952#issuecomment-1153339532)
> 
> Please can you try the latest version of besu

I also commented on #3952 ([comment](https://github.com/hyperledger/besu/issues/3952#issuecomment-1153571075)) and pointed out that on the recent version, the `bin/besu` command has been moved somewhere else.

That's the reason I had to use an older version.

**Comment by usmansaleem** [comment]
Closed due to #3952
